### ISSUE TITLE
Lutris's GUI doesn't start on Fedora32

### ISSUE THREAD

**Issue by v21cesc** [lutris/lutris#2656]
console log as follows

lutris
2020-03-14 15:32:44,492: Running Lutris 0.5.4
2020-03-14 15:32:44,492: Using NVIDIA drivers 440.64 for x86_64
2020-03-14 15:32:44,493: GPU: GeForce GTX 1660 Ti
2020-03-14 15:32:44,493: GPU: 10DE:2182 1462:8D90 using nvidia drivers
2020-03-14 15:32:44,568: Vulkan is supported
2020-03-14 15:32:44,569: Updating DXVK versions
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/lutris/gui/application.py", line 173, in do_activate
    self.window = LutrisWindow(application=self)
  File "/usr/lib/python3.8/site-packages/lutris/gui/lutriswindow.py", line 102, in __init__
    self.game_store = self.get_store()
  File "/usr/lib/python3.8/site-packages/lutris/gui/lutriswindow.py", line 315, in get_store
    game_store = GameStore(
  File "/usr/lib/python3.8/site-packages/lutris/gui/views/store.py", line 133, in __init__
    self.modelsort = Gtk.TreeModelSort.sort_new_with_model(self.modelfilter)
AttributeError: type object 'TreeModelSort' has no attribute 'sort_new_with_model'
2020-03-14 15:33:00,474: Updating D9VK versions

**Comment by v21cesc** [comment]
The issue is still there after deleting lutris’s config files

**Comment by Feichtmeier** [comment]
Same problem on Ubuntu focal fossa daily, lutris crashes after start:

```
2020-03-14 21:23:56,263: Running Lutris 0.5.4
2020-03-14 21:23:56,264: Using Intel
2020-03-14 21:23:56,264: Running Mesa driver 20.0.0 on Mesa Intel(R) UHD Graphics 620 (WHL GT2) (0x3ea0)
2020-03-14 21:23:56,264: GPU: 8086:3EA0 1028:08AF using i915 drivers
2020-03-14 21:23:56,312: Vulkan is supported
2020-03-14 21:23:56,313: Updating DXVK versions
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/lutris/gui/application.py", line 173, in do_activate
    self.window = LutrisWindow(application=self)
  File "/usr/lib/python3/dist-packages/lutris/gui/lutriswindow.py", line 102, in __init__
    self.game_store = self.get_store()
  File "/usr/lib/python3/dist-packages/lutris/gui/lutriswindow.py", line 315, in get_store
    game_store = GameStore(
  File "/usr/lib/python3/dist-packages/lutris/gui/views/store.py", line 133, in __init__
    self.modelsort = Gtk.TreeModelSort.sort_new_with_model(self.modelfilter)
AttributeError: type object 'TreeModelSort' has no attribute 'sort_new_with_model'
2020-03-14 21:23:57,283: Updating D9VK versions
```

**Comment by connectety** [comment]
Not a Maintainer or Contributer or anything,
but had a quick look at it.
This seams to be caused by some an API-change in a GTK+ in 3.24.12 / 3.24.13 (version number is taken from a gentoo bugzilla).
This also seams to be fixed in 12a458db3c590c3f1972594e091a24cb4d65b610.

**Comment by v21cesc** [comment]
> Not a Maintainer or Contributer or anything, but had a quick look at it.
> This seams to be caused by some an API-change in a GTK+ in 3.24.12 / 3.24.13 (version number is taken from a gentoo bugzilla).
> This also seams to be fixed with the commit [12a458d](https://github.com/lutris/lutris/commit/12a458db3c590c3f1972594e091a24cb4d65b610).

It works
